pep-0657.rst

```diff
@@ -191,11 +191,16 @@ this PEP:
   four-element tuples containing the full location of every instruction
   (including start line, end line, start column offset and end column offset)
   or ``None`` if the code object was created without the offset information.
-* Three new C-API functions, ``PyCode_Addr2EndLine``, ``PyCode_Addr2StartOffset``
-  and ``PyCode_Addr2EndOffset`` will be added that can obtain the end line, the
-  start column offsets and the end column offset respectively given the index
-  of a bytecode instruction. These functions will return 0 if the information
-  is not available.
+* One new C-API function: ::
+
+    int PyCode_Addr2Location(
+        PyCodeObject *co, int addr,
+        int *startline, int *startcolumn,
+        int *endline, int *endcolumn)
+
+  will be added so the end line, the start column offsets and the end column
+  offset can be obtained given the index of a bytecode instruction. This
+  function will set the values to 0 if the information is not available.
```

Comment:
> This function will set the values to 0 if the information is not available.

If so, do we need the return type of `int`?

Reply:
Yes, that is to indicate that somethign went wrong, while setting the values to 0 means "this particular one is missing". The second case is not an error